File: api/php/runner_velocities.php
<?php
   
    include_once('creds.php'); // (added to gitignore - see creds_example.php)
    echo (isset($k) && isset($_GET['sc'])) ? file_get_contents("https://www.tpd.zone/json-rpc/v2/vendors/runner_velocities/?sc={$_GET['sc']}&includes={$_GET['includes']}&source={$_GET['source']}&k={$k}") : [];

File: api/py/runner_velocities.py
import requests

try:
    from creds import k
except ImportError:
    k = None

if k and 'sc' in request.args and 'includes' in request.args:
    response = requests.get(f"https://www.tpd.zone/json-rpc/v2/vendors/runner_velocities/?sc={request.args['sc']}&includes={request.args['includes']}&source={request.args['source']}&k={k}")
    print(response.content)
else:
    print([])